hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityAutoConfiguration.java

```diff
@@ -82,4 +88,33 @@ public SecurityTokenGenerator securityTokenGenerator() {
         return new SecurityTokenGenerator();
     }
 
+    /**
+     * @return {@link LogoutHandler} bean
```

Comment:
change the logoutHandler to AuthentiationSuccessHandler

Reply:
done